### PR TITLE
Feature/iot arduino code

### PR DESCRIPTION
**Pull request by Hunter27**
created multiple sensor functionality and basic fault tolerance procedures

### CODE REVIEW

File: CheckValues/Program.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CheckValues
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
```
Comment (CilliersN) [inline CheckValues/Program.cs]: what??

Comment (ash-rabbit) [inline CheckValues/Program.cs]: Why?

**Review by ash-rabbit — CHANGES_REQUESTED** [review]

File: CheckValues/Program.cs (hunk 1 above)
Comment (grootendorste) [inline CheckValues/Program.cs]: Hello

Comment (Hunter27) [reply CheckValues/Program.cs]: man i didnt even know that was there

Comment (Hunter27) [reply CheckValues/Program.cs]: We use that project for debugging

File: CheckValues/Results.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+namespace CheckValues
+{
+    public class Results
+    {
+        public MonitorsController getController()
+        {
+            string connection = "";
+            IConfiguration chosenconfig;
+            var jsonconfig = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json")
```
Comment (grootendorste) [inline CheckValues/Results.cs:23]: I don't think this is correct, usually there's a tab when calling more functions on an object.

File: CheckValues/Program.cs (hunk 1 above)
Comment (Hunter27) [reply CheckValues/Program.cs]: guys I didnt know that was there

Comment (CilliersN) [reply CheckValues/Program.cs]: dont you read your pr when creating it ?

**Review by CilliersN — CHANGES_REQUESTED** [review]

File: CheckValues/Results.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        public int getLen()
+        {
+            MonitorsController _controller = getController();
+            Task<ActionResult<IEnumerable<MonitorsEntry>>> item = _controller.Get();
+            ActionResult<IEnumerable<MonitorsEntry>> item2 = item.Result;
+            IEnumerable<MonitorsEntry> item3 = item2.Value;
+            int count = 0;
+            foreach (MonitorsEntry itemv in item3)
+            {
+                count++;
```
Comment (grootendorste) [inline CheckValues/Results.cs:66]: Can't you do a .Count function on item3 to get the count of the item?

File: CheckValues/Results.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            Task<ActionResult<MonitorsEntry>> item = _controller.Get(1);
+            ActionResult<MonitorsEntry> item2 = item.Result;
+            MonitorsEntry item3 = item2.Value;
+            Console.WriteLine(item3.Id);
+
+        }
+        public int getLen()
+        {
+            MonitorsController _controller = getController();
+            Task<ActionResult<IEnumerable<MonitorsEntry>>> item = _controller.Get();
```
Comment (grootendorste) [inline CheckValues/Results.cs:60]: All these can be changed to vars.

File: CheckValues/Program.cs (hunk 1 above)
Comment (Hunter27) [reply CheckValues/Program.cs]: took it out bro

File: WaterLog_Backend/Properties/launchSettings.json.orig [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+<<<<<<< HEAD
```
Comment (ash-rabbit) [inline WaterLog_Backend/Properties/launchSettings.json.orig:6]: you should not commit this file plus there are conflicts

File: WaterLog_Backend/Controllers/ReadingsController.cs.orig [hunk 6]
```diff
@@ -0,0 +1,93 @@
+﻿using System;
```
Comment (CilliersN) [inline WaterLog_Backend/Controllers/ReadingsController.cs.orig:1]: why are you comitting an orig file ?

**Review by ash-rabbit — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Controllers/ReadingsController.cs.orig (hunk 6 above)
Comment (grootendorste) [inline WaterLog_Backend/Controllers/ReadingsController.cs.orig:1]: Did you commit an orig file?

**Review by grootendorste — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Properties/launchSettings.json.orig [hunk 7]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (CilliersN) [inline WaterLog_Backend/Properties/launchSettings.json.orig:1]: remove this file

**Pull request closed without merging**